soc/arm/arm/mps2/dts_fixup.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* SoC level DTS fixup file */
+
```

Comment:
This needs an `#if defined(CONFIG_SOC_MPS2_AN521)` so we don't break MPS2 AN385.